Code/GraphMol/Descriptors/BCUT.cpp

```diff
@@ -101,10 +105,14 @@ std::pair<double, double> BCUT2D(const ROMol &m,
 }
 
 std::vector<double> BCUT2D(const ROMol &m) {
+  unsigned int num_atoms = m.getNumAtoms();
+  if(num_atoms == 0) {
+        return {0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0};
```

Comment:
there's something about this list of 0's that makes me nervous.
How about something like this (not tested)?
```suggestion
        constexpr size_t bcutSize=8;
        return std::vector<double>(8,0.0);
```

Reply:
This is what I use in the test, so it's fine.  I might make the 8 a non-magic number though.

Reply:
done